.gitignore

```diff
@@ -158,3 +158,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# mkdocs generated documentation
+docs/generated/
```

Comment:
I might have duplicated this on the other PR. I'll make sure I remove it from mine

Reply:
Shouldn't make a difference, we should be able to add it in both. (And I would, since this is necessary for both).